features/step_definitions/user_steps.rb

```diff
@@ -47,9 +47,16 @@
   @user.update(reset_password_sent_at: Time.zone.now)
 end
 
-When(/^I choose a photo file named "([^"]*)" to upload$/) do | filename |
-  page.attach_file('user_member_photo',
+When(/^I choose a "([^"]*)" file named "([^"]*)" to upload$/) do | fieldname, filename |
+  page.attach_file(fieldname,
                    File.join(Rails.root, 'spec', 'fixtures',
                              'member_photos', filename), visible: false)
   # ^^ selenium won't find the upload button without visible: false
 end
+
+When(/^I choose an SHF "([^"]*)" file named "([^"]*)" to upload$/) do | fieldname, filename |
```

Comment:
To me, "SHF" has an initial _sound_ of a vowel, not a consonant.  (Because "S" sounds like it is spelled ' _ess_).

https://www.quickanddirtytips.com/education/grammar/a-versus-an